admin/commands/inventory/add_agent_external_exporter.go

```diff
@@ -48,19 +48,7 @@ func (res *addAgentExternalExporterResult) String() string {
 	return commands.RenderTemplate(addAgentExternalExporterResultT, res)
 }
 
-type addAgentExternalExporterCommand struct {
-	RunsOnNodeID string
-	ServiceID    string
-	Username     string
-	Password     string
-	CustomLabels string
-	Scheme       string
-	MetricsPath  string
-	ListenPort   int64
-	PushMetrics  bool
-}
-
-func (cmd *addAgentExternalExporterCommand) Run() (commands.Result, error) {
+func (cmd *ExternalExporterCommand) RunCmd() (commands.Result, error) {
```

Comment:
let's keep struct names the same
```suggestion
func (cmd *AddExternalExporterCommand) RunCmd() (commands.Result, error) {
```